packages/widgetbook_generator/test/src/tree/tree_test.dart

```diff
@@ -25,11 +27,11 @@ void main() {
       final useCases = [
         MockUseCaseMetadata(
           componentName: 'Alpha',
-          componentImportUri: 'package:foo/widgets/alpha/alpha.dart',
+          pathInWidgetbook: 'widgets/alpha',
         ),
         MockUseCaseMetadata(
           componentName: 'Beta',
-          componentImportUri: 'package:foo/widgets/beta/beta.dart',
+          pathInWidgetbook: 'widgets/beta',
```

Comment:
Let's keep this using `componentImportUri` instead of `navPath` (`pathInWidgetbook`) to make sure that both flows work properly.

Reply:
I wanted to do that initially, however by my understanding, the UseCaseMetadata contains already parsed information. For generating the tree, only `navPath` is being used, because this is one step after the first pass of the generator, or am I missing anything?

Reply:
Yeah, you are totally right about this one.